src/save.js

```diff
@@ -0,0 +1,89 @@
+const Promise = require('bluebird'),
+	github = require('octonode'),
+	converter = require('biblatex-csl-converter'),
+	token = process.env.GIT_TOKEN;
+
+class GithubLatexGateway {
+	constructor(config){
+		"use strict";
+		if (!(config && config.user && config.path && config.repo)) throw new Exception("Config is missing or does not contain required fields.");
```

Comment:
require at least user, repo and filepath for ex-/import. For access of third person's repos also add owner.

Once test is over, authentication object must be added